rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -1385,7 +1385,7 @@ error_code cellCameraReadEx(s32 dev_num, vm::ptr<CellCameraReadEx> read)
 
 	if (has_new_frame)
 	{
-		g_camera.frame_timestamp = (get_guest_system_time() - g_camera.start_timestamp);
+		g_camera.frame_timestamp = ::narrow<u32>(get_guest_system_time() - g_camera.start_timestamp);
```

Comment:
This can crash, do not rely on time consistency, that is why many games on rpcs3 fail to boot.

Reply:
I want to know it though. why should i allow such nonsense?

Reply:
What if I pause rpcs3 in an external debugger such as vusual studio, why should it crash then? that is nonsense.

Reply:
why would it go above u32 ?

Reply:
1 hour.

Reply:
you can play a thousand hours before you reach that number

Reply:
Right, I was totally wrong, I can spend another 11 minutes after that hour before it crashes. My mistake, 11 minutes is a huge difference.

Reply:
is system_time_t ms or us ?

Reply:
seems like a bug

Reply:
us.

Reply:
why is it s64 then and not u64?

Reply:
I can't really browse the code thoroughly right now.

Reply:
you'd have to browse the sdk though. I don't think ours is correct. it's only mentioned once and is unused anyway.
But I assume many places use s64

Reply:
I changed all occasions to u64... s64 only confuses for no reason.